lte/gateway/Vagrantfile

```diff
@@ -74,8 +78,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # using a specific IP.
     magma_trfserver.vm.network "private_network", ip: "192.168.60.144", nic_type: "82540EM"
     # iperf3 server IP.
-    magma_trfserver.vm.network "private_network", ip: "192.168.129.42", nic_type: "82540EM"
-    magma_trfserver.vm.network "private_network", ip: "3001::2", nic_type: "82540EM"
+    # Set to an internal network to prevent possible connection to vagrant host instead of other VM
+    # For IPv6 network,
+    # - `type` specified due to https://github.com/hashicorp/vagrant/issues/12839
+    # - `netmask` specified to enforce correct mask when using internal network
+    magma_trfserver.vm.network "private_network", ip: "192.168.129.42", nic_type: "82540EM", virtualbox__intnet: true
```

Comment:
You want these networks to have different names not just true.

As show in the second example here:
https://developer.hashicorp.com/vagrant/docs/providers/virtualbox/networking#virtualbox-internal-network

Reply:
Otherwise I think they will all end up in the same virtualbox network

Reply:
Done